backend/danswer/dynamic_configs/users.py

```diff
@@ -0,0 +1,21 @@
+from typing import cast
```

Comment:
Slight preference to have dynamic_configs only contain the interfaces/implementations of the storage option and not add specific feature functionality into here